### PR TITLE
Create spiral_print.java

### PR DESCRIPTION
**Pull request by Arpit2401**

### CODE REVIEW

**Comment by Arpit2401** [comment]
I am a GSSOC'21 participant and made this pull request. Please accept it amd merge it. Thanks!

File: Code/Java/spiral_print.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import java.util.*;
```
Comment (raksha009) [inline Code/Java/spiral_print.java:1]: Add a short description of the problem statement.

File: Code/Java/spiral_print.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    		}
+    		sc++;
+    	}
+    }
+
+}
+
+
+/* 
+	Test Cases :
```
Comment (raksha009) [inline Code/Java/spiral_print.java:72]: add 2 test cases minimum

File: Code/Java/spiral_print.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    
+    		// first coloumn
+    		for(int i=er;i>=sr;i--){
+                System.out.print(a[i][sc] + " ");
+    			ele++;
+    		}
+    		sc++;
+    	}
+    }
+
```
Comment (raksha009) [inline Code/Java/spiral_print.java:67]: Remove extra line

**Review by raksha009 — CHANGES_REQUESTED** [review]
Plz follow consistent indentation throughout.

**Comment by Arpit2401** [comment]
I have updated spiral_print.java with the required changes as asked by @raksha009. Do review it. Thanks!

File: Code/Java/spiral_print.java [hunk 4]
```diff
@@ -1,78 +1,76 @@
...
+			for(int j =0; j < c; j++){
+  			a[i][j]=sc.nextInt();
+			}
+		}
+		spiralprint(a,r,c);
+	}
+	
+	public static void spiralprint(int a[][], int r, int c){
+		//start row
+    int sr = 0;
```
Comment (raksha009) [inline Code/Java/spiral_print.java:20]: Inconsistent indentation.

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by Arpit2401** [comment]
Indentation problem resolved. Do review it @raksha009

**Review by raksha009 — APPROVED** [review]

**Comment by Arpit2401** [comment]
@tarun26091999 sir please review my pr and do let me know if there are any changes.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
why you are trying to merge codes of spiral print and wave print by same PR. also issue is not linked to your PR.

**Comment by Arpit2401** [comment]
Sorry, @tarun26091999 sir wave print program was unknowingly added to this PR. I have removed that file. Also, my issue related to this PR is #616. Do let me know what has to be done now as I am a beginner in open source.

**Comment by tarun26091999** [comment]
you cannot raise a PR until the issue is assigned to you.
